### PR TITLE
Add desc for components

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock <sherlockxu@yunify.com>

Add desc for components

### CODE REVIEW

**Comment by Sherlock113** [comment]
@FeynmanZhou Please have a look at this. Thanks.

File: deploy/cluster-configuration.yaml [hunk 1]
```diff
@@ -46,18 +46,18 @@ spec:
...
+  monitoring:  # This feature gives visibility into the amount of compute resources (e.g. CPU and memory utilization) available in the workspace and allow you to track usage at different levels.
     prometheusReplicas: 1
     prometheusMemoryRequest: 400Mi
     prometheusVolumeSize: 20Gi
     alertmanagerReplicas: 1
-    notification:
+    notification: # Email notifications are supported, and users can customize the sender email address and test the email server before it is put into use.
       enabled: false
-  multicluster:
+  multicluster:  # Users can create projects to run across clusters, building a container environment for rapid iteration of applications and achieve high availability.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:58]: A multi-cluster control plane that helps you to centrally manage multiple K8s clusters from different cloud providers, it supports distribute apps across multiple clusters and cloud providers.

File: deploy/cluster-configuration.yaml [hunk 2]
```diff
@@ -46,18 +46,18 @@ spec:
...
+  metrics_server:  # Users can install metrics-server to enable HPA (Horizontal Pod Autoscaler) which automatically scales the number of pods, deployments, or statefulsets based on observed CPU utilization.
     enabled: false
-  monitoring:
+  monitoring:  # This feature gives visibility into the amount of compute resources (e.g. CPU and memory utilization) available in the workspace and allow you to track usage at different levels.
     prometheusReplicas: 1
     prometheusMemoryRequest: 400Mi
     prometheusVolumeSize: 20Gi
     alertmanagerReplicas: 1
-    notification:
+    notification: # Email notifications are supported, and users can customize the sender email address and test the email server before it is put into use.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:56]: It supports manages notifications in multi-tenant Kubernetes clusters. It receives alerts or notifications from different senders and then sends notifications to various tenant receivers

File: deploy/cluster-configuration.yaml [hunk 3]
```diff
@@ -46,18 +46,18 @@ spec:
     elasticsearchDataVolumeSize: 20Gi
     logMaxAge: 7
     elkPrefix: logstash
-  metrics_server:
+  metrics_server:  # Users can install metrics-server to enable HPA (Horizontal Pod Autoscaler) which automatically scales the number of pods, deployments, or statefulsets based on observed CPU utilization.
     enabled: false
-  monitoring:
+  monitoring:  # This feature gives visibility into the amount of compute resources (e.g. CPU and memory utilization) available in the workspace and allow you to track usage at different levels.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:51]: With multi-dimensional monitoring metrics, the built-in monitoring dashboard gives visibility into computing resources ranging from infrastructure to applications.

File: deploy/cluster-configuration.yaml [hunk 4]
```diff
@@ -23,21 +23,21 @@ spec:
   console:
     enableMultiLogin: false  # enable/disable multi login
     port: 30880
-  alerting:
+  alerting: # Users can customize alerting policies to send messages to users in time with different time intervals and alerting levels to choose from. 
     enabled: false
   auditing:
     enabled: false
-  devops:
+  devops:  # A one-stop DevOps console is provided to enable developers to create, build, test and release their applications in an automated way.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:30]: A one-stop DevOps console with Jenkins as its engine is provided to enable developers to create, build, test and release their applications in an automated way.

File: deploy/cluster-configuration.yaml [hunk 5]
```diff
@@ -23,21 +23,21 @@ spec:
...
     enabled: false
     jenkinsMemoryLim: 2Gi
     jenkinsMemoryReq: 1500Mi
     jenkinsVolumeSize: 8Gi
     jenkinsJavaOpts_Xms: 512m
     jenkinsJavaOpts_Xmx: 512m
     jenkinsJavaOpts_MaxRAM: 2g
-  events:
+  events:  # Events that happen inside a cluster are recorded, such as container running status, image pulling failures and node scheduling activities.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:38]: It provides a graphical web console for Kubernetes Events exporting, filtering and alerting in multi-Tenant Kubernetes clusters.

File: deploy/cluster-configuration.yaml [hunk 6]
```diff
@@ -23,21 +23,21 @@ spec:
...
     enabled: false
-  logging:
+  logging:  # Flexible logging functions are provided for query, collection and management. Additional log collectors can be added, such as Elasticsearch, Kafka and Fluentd.
```
Comment (FeynmanZhou) [inline deploy/cluster-configuration.yaml:40]: Flexible logging functions are provided for the log query, collection, and management in a unified console. Additional log collectors can be added, such as Elasticsearch, Kafka, and Fluentd.

**Pull request closed without merging**